src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@thinkingmedia I don't think __toString() allows to throw an error even if you are using PHP7. Try the following code on PHP7:
```php
class X {
   public function __toString() {
      throw new Error('message');
   }
}

try {
    echo new X;
} catch (Error $e) {
    // If you throw Error from whitn __toString(), PHP7 throws an uncatchable fatal error.
    // So never come here.
    var_dump("Never come here");
}
```
You would get an uncatchable fatal error.

So I am not sure why your code solves your issue. If you can catch an Error object in __toString(), Cake's error handler should be able to catch the Error object without create and throw a new Error object because both thrown objects are an instance of Error, aren't they? Your change seems weird to me. It seems to be like the following:
```php
// Before
public function __toString() {
    throw new Error('message');
}

// After
public function __toString() {
    try {
        throw new Error('message');
    } catch (Error $e) {
        throw new Error($e->getMesage());
    }
}
```
Could you please explain why/how your code works? I am still guessing that you (or Cake?) catch the Error object in render() and return null.